.github/workflows/ci.js.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: CI
+
+on:
+  push:
+    branches: ["master"]
```

Comment:
Why only master?

Reply:
we can set up only `on: push`. Otherwise on pull requests branch is built twice (on push + on PR)

Reply:
Yes, but I supposed, that we would add not only master

Reply:
i'm not sure what you're suggesting

Reply:
Vlad and I suggest all the branches, not only master

Reply:
sorry, i don't see any advantages of running tests on every push in any branch. let's stick to the defaults just for now: https://github.com/actions/starter-workflows/blob/main/ci/node.js.yml we may alter this later